src/IssueViz.Security/OpenInIDE/Http/StatusOwinRequestHandler.cs

```diff
@@ -45,10 +45,11 @@ internal StatusOwinRequestHandler(IStatusRequestHandler statusRequestHandler)
 
         public async Task ProcessRequest(IOwinContext context)
         {
+            context.Response.Headers["Access-Control-Allow-Origin"] = context.Request.Headers["Origin"];
```

Comment:
@jblievremont is there a reason to do it before processing the request? why is it not next to ContentType assignment?

Reply:
Yup, I moved it to `OwinPipelineProcessor` - thanks for the pointer! The idea is that we want this to be done as early as possible in the request handling, for the browser to allow the frontend code to see the response even in case of a failure.